.gitignore

```diff
@@ -15,3 +15,4 @@ Project_2/Java_Part_1/.project
 Project_2/Java_Part_1/bin/pom.xml
 Project_2/Java_Part_1/bin/.project
 Project_2/Java_Part_1/.settings/org.eclipse.jdt.core.prefs
+.vscode/settings.json
```

Comment:
next time please include all visual studio project files

Reply:
Figured it was better to not given the different env. setups since it was causing problems for us in our capstone. I'll make sure to do that onwards.